lib/datadog/profiling/component.rb

```diff
@@ -97,7 +98,7 @@ def self.build_profiler_component(settings:, agent_settings:, optional_tracer:)
           load_pprof_support
 
           recorder = build_profiler_old_recorder(settings)
-          collector = build_profiler_oldstack_collector(settings, recorder, optional_tracer)
+          worker = build_profiler_oldstack_collector(settings, recorder, optional_tracer)
```

Comment:
Should we rename `build_profiler_oldstack_collector` to `build_profiler_oldstack_worker`?

Reply:
I don't think it's worth it -- the new profiler has been released, and on by default for almost two quarters.

I was keeping the old code around as a safety policy against "unknown-unknowns", e.g. a customer running into an issue where the new profiler really couldn't work for them, but after almost two quarters, this did not happen and the only reason this code hasn't yet been deleted is time ;) 

So TL;DR: I don't think it's worth spending any more time on code that's about to be deleted.